packages/app/src/components/SQLInlineEditor.tsx

```diff
@@ -359,12 +359,16 @@ export function SQLInlineEditorControlled({
 }: Omit<SQLInlineEditorProps, 'value' | 'onChange'> & UseControllerProps<any>) {
   const { field } = useController(props);
 
+  // Guard against wrongly typed values
+  const value = field.value || props.defaultValue;
+  const stringValue = typeof value === 'string' ? value : '';
```

Comment:
we can merge this as-is, would it be appropriate to throw a warn here to understand when the component is being misused or is this largely doing null checks and the like?

Reply:
this is mainly for the `select` field as it can be array. I added a console.error to make sure we get notified if that happens again